src/System.IO.Compression/tests/ZipArchive/zip_InvalidParametersAndStrangeFiles.cs

```diff
@@ -126,6 +130,370 @@ public static async Task ZipArchiveEntry_InvalidUpdate(string zipname)
             }
         }
 
+        [Theory]
+        [InlineData("normal.zip")]
```

Comment:
Why a theory with only one input? That seems easier as a `[Fact]`

Reply:
left over from copy paste existing test, will update